README.md

```diff
@@ -1,22 +1,57 @@
 # wag
 sWAGger - Web API Generator
 
-# Running
+## Usage
+### Generating Code
+Create a swagger.yml file with your [service definition](http://editor.swagger.io/#/). Note that WAG supports a [subset](https://github.com/Clever/wag#swagger-spec) of the Swagger spec. Then generate your code:
 ```
-go run main.go genclients.go
-cd impl/ && ./impl
+make swagger
```

Comment:
@kvigen mind if I edit this directly to show how to set up your makefile targets and utilize swagger.mk?